src/SourceBuild/content/build.proj

```diff
@@ -25,6 +27,20 @@
              StopOnFirstFailure="true" />
   </Target>
 
+  <Target Name="PublishVerticalManifest" AfterTargets="Build">
+    <PropertyGroup>      
+      <VerticalAssetManifestLocalPath>$(ArtifactsDir)VerticalManifest.xml</VerticalAssetManifestLocalPath>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <AssetManifestPaths Include="$(AssetManifestsIntermediateDir)\**\*.xml" />
```

Comment:
nit: while Arcade unfortunately doesn't follow the convention in some places, it would be good to be consistent here.

MSBuild items are usually singular not plural, i.e. `Compile`, `None`, `Content`, `Reference`, etc.

```suggestion
      <AssetManifestPath Include="$(AssetManifestsIntermediateDir)\**\*.xml" />
```